Classes/Controller/TeaController.php

```diff
@@ -23,13 +24,15 @@ public function injectTeaRepository(TeaRepository $teaRepository): void
         $this->teaRepository = $teaRepository;
     }
 
-    public function indexAction(): void
+    public function indexAction(): ResponseInterface
```

Comment:
We should also have tests that the actions return a `HtmlResponse` with the content from `htmlResponse`.

Reply:
Done